pkgs/by-name/vo/voikko-fi/package.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenvNoCC,
+  python3,
+  foma,
+  libvoikko,
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "voikko-fi";
+  version = "2.5";
+
+  src = fetchFromGitHub {
+    owner = "voikko";
+    repo = "corevoikko";
+    tag = "rel-voikko-fi-${finalAttrs.version}";
+    hash = "sha256-0MIQ54dCxyAfdgYWmmTVF+Yfa15K2sjJyP1JNxwHP2M=";
+  };
```

Comment:
Hot take (probably disagreeable with other maintainers): I'd be happy with `version = "0.0.0"` and `inherit (libvoikko) src`, and with 1 FOD instead of 2 FODs. I don't expect anyone to run `lib.version*` comparisons on this package, and I think it's reasonable to expect that `libvoikko` and `voikko-fi` taken from the same commit are compatible... `libvoikko` also seems to have more frequent releases than `voikko-fi` anyway. Also I wonder if rryantm is going to be able to distinguish between the voikko-fi tags and libvoikko tags.

But I'll leave that decision to you